### PR TITLE
Rename Google auth endpoint parameter approval_prompt to prompt

### PR DESCRIPTION
**Pull request by troydai**
Issue #971

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs [hunk 1]
```diff
@@ -87,18 +87,19 @@ public GoogleHandler(HttpClient httpClient)
...
             queryStrings.Add("response_type", "code");
             queryStrings.Add("client_id", Options.ClientId);
             queryStrings.Add("redirect_uri", redirectUri);
 
-            AddQueryString(queryStrings, properties, "scope", scope);
-
+            AddQueryString(queryStrings, properties, "scope", FormatScope());
             AddQueryString(queryStrings, properties, "access_type", Options.AccessType);
-            AddQueryString(queryStrings, properties, "approval_prompt");
+            AddQueryString(queryStrings, properties, "prompt");
```
Comment (PinpointTownes) [inline src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs]: Isn't that a functional breaking change, as it changes the authentication property name?

Comment (PinpointTownes) [inline src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs]: Note: you'll also have to fix the Google tests: https://github.com/aspnet/Security/blob/dev/test/Microsoft.AspNetCore.Authentication.Test/Google/GoogleMiddlewareTests.cs

Comment (troydai) [inline src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs]: True. I'll keep both.

Comment (troydai) [inline src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs]: Tests are updated.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**